src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 1) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return days * 40;
```

Comment:
You use the expression: `(days * 40`) in other places, so let's create a separate variable for this.